browser/help/topics/v3-ht-annotations.md

```diff
@@ -165,22 +165,22 @@ The gnomAD v3 Hail Table annotations are defined below:
+  - **revel**: Rare Exome Variant Ensemble Learner (REVEL) [score](https://sites.google.com/site/revelgenomics/about). An ensemble score for predicting the pathogenicity of missense variants (based on 13 other variant predictors).
+    - **revel_max**: Score predicting the pathogenicity of missense variants with a range of 0 to 1. Variants with higher scores are predicted to be more likely to be deleterious. We prioritize max scores for MANE Select transcripts where possible and otherwise report a score for the canonical transcript.
+  - **primateai_3d**: A deep neural network-trained [score](https://www.science.org/doi/10.1126/science.abn8197) for classifying pathogenicity of missense variants.
+    - **primateai-3d_max**: Score ranging from 0 to 1 predicting the deleteriousness of missense variants. Higher scores indicate a variant is more likely to be deleterious. We prioritize max scores for MANE Select transcripts where possible and otherwise report a score for the canonical transcript. 
+  - **cadd**: Combined Annotation Dependent Deletion [score](https://cadd.gs.washington.edu/) used to predict deleteriousness of SNVs and indels.
+    - **phred**: CADD Phred-like scaled C-scores ranging from 1 to 99 based on the rank of each variant relative to all possible 8.6 billion substitutions in the human reference genome. Larger values indicate increased predicted deleteriousness.
+    - **raw_score**: Unscaled CADD scores indicating whether a variant is likely to be "observed" (negative values) vs "simulated" (positive values). Larger values indicate increased predicted deleteriousness.
+  - **sift**: Sorting Tolerant From Intolerant (SIFT) [score](https://www.nature.com/articles/nprot.2009.86) used to predict the impact of SNVs on protein function.
+    - **sift_max**: Scaled probability of the amino acid substitution being tolerated, ranging from 0 to 1. Scores below 0.05 are predicted to impact protein function.  A normalized probability of observing the new amino acid at that position to affect protein function, and ranges from 0 to 1. We prioritize max scores for MANE Select transcripts where possible and otherwise report a score for the canonical transcript. 
+  - **polyphen**: Polymorphism Phenotyping v2 (Polyphen-v2 [score](https://www.nature.com/articles/nmeth0410-248)) that predicts the impact of an SNV on protein structure and function.
+    - **polyphen_max**: Prediction of the possible impact of an amino acid substitution on the structure and function of a human protein, ranging from 0.0 (tolerated) to 1.0 (deleterious).  We prioritize max scores for MANE Select transcripts where possible and otherwise report a score for the canonical transcript.
```

Comment:
```suggestion
    - **polyphen_max**: Prediction of the possible impact of an amino acid substitution on the structure and function of a human protein, ranging from 0.0 (tolerated) to 1.0 (deleterious). We prioritize max scores for MANE Select transcripts where possible and otherwise report a score for the canonical transcript.
```